docs/tutorial/chapter-3-adding-mutations-to-your-api.md

```diff
@@ -423,7 +423,8 @@ mutation {
 
 ## Wrapping Up
 
-Congratulations! You can now read and write to your API. Good job. But, so far you've been validating your work by manually interacting with the Playground. That may be reasonable at first (depending on your relationship to TDD) but it will not scale. At some point you are going to want automated testing. Nexus takes testing seriously and in the next chapter we'll show you how. See you there!
+Congratulations! You can now read and write to your API.
```

Comment:
Was the goal here to have two separate paragraphs?

If so to achieve that I added an extra line break and new css rules to support multple paragraphs trailing wrapping up header. Looks like this:

![image](https://user-images.githubusercontent.com/284476/83633575-2a6d9900-a56f-11ea-949f-0a406f7d84f7.png)

Reply:
Ah indeed, thanks!